src/components/form.jsx

```diff
@@ -0,0 +1,61 @@
+export default function Form({ handleSubmit, form, onChange }) {
+	return (
+		<>
+			<form onSubmit={handleSubmit} className="">
```

Comment:
Nice work using the `onSubmit` handler for the form submission! Using the `enter` key to submit works great